kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/ProcessServicesClientImpl.java

```diff
@@ -329,6 +330,35 @@ public Long startProcess(String containerId, String processId, Map<String, Objec
     }
 
 
+    @Override
+    public Map<String, Object> startSynchronousProcess(String containerId, String processId, Map<String, Object> variables) {
+        Object result = null;
+
+        if( config.isRest() ) {
+
+            Map<String, Object> valuesMap = new HashMap<String, Object>();
```

Comment:
Not needed to declare types
```suggestion
            Map<String, Object> valuesMap = new HashMap<>();
```

Reply:
done